src/app/components/feed/SaveFeed.module.css

```diff
@@ -18,10 +18,14 @@
   flex-direction: column;
   gap: 16px;
   height: fit-content;
-  max-width: 650px;
 }
 
 .saveFeedContentNarrow {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  max-width: 350px;
```

Comment:
the more "correct" flexbox-y way of doing this, would be to remove the max and min widths and go with `flex: 0 0 350px;`
which translates to, don't grow, don't shrink, be 350px wide